frontend/templates/admin/dashboard.html

```diff
@@ -0,0 +1,138 @@
+            clip-rule="evenodd"
+          />
+        </svg>
+        <span>Users</span>
+      </a>
+      <a href="/events">
+        <svg
+          class="h-6 w-6"
+          aria-hidden="true"
+          xmlns="http://www.w3.org/2000/svg"
+          width="24"
+          height="24"
+          fill="currentColor"
+          viewBox="0 0 24 24"
+        >
+          <path
+            fill-rule="evenodd"
+            d="M6 5V4a1 1 0 1 1 2 0v1h3V4a1 1 0 1 1 2 0v1h3V4a1 1 0 1 1 2 0v1h1a2 2 0 0 1 2 2v2H3V7a2 2 0 0 1 2-2h1ZM3 19v-8h18v8a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2Zm5-6a1 1 0 1 0 0 2h8a1 1 0 1 0 0-2H8Z"
+            clip-rule="evenodd"
+          />
+        </svg>
+        <span>Events</span>
+      </a>
+      <a href="/posts">
+        <svg
+          class="h-6 w-6"
+          aria-hidden="true"
+          xmlns="http://www.w3.org/2000/svg"
+          width="24"
+          height="24"
+          fill="currentColor"
+          viewBox="0 0 24 24"
+        >
+          <path
+            fill-rule="evenodd"
+            d="M5 3a2 2 0 0 0-2 2v14a2 2 0 0 0 2 2h11.5c.07 0 .14-.007.207-.021.095.014.193.021.293.021h2a2 2 0 0 0 2-2V7a1 1 0 0 0-1-1h-1a1 1 0 1 0 0 2v11h-2V5a2 2 0 0 0-2-2H5Zm7 4a1 1 0 0 1 1-1h.5a1 1 0 1 1 0 2H13a1 1 0 0 1-1-1Zm0 3a1 1 0 0 1 1-1h.5a1 1 0 1 1 0 2H13a1 1 0 0 1-1-1Zm-6 4a1 1 0 0 1 1-1h6a1 1 0 1 1 0 2H7a1 1 0 0 1-1-1Zm0 3a1 1 0 0 1 1-1h6a1 1 0 1 1 0 2H7a1 1 0 0 1-1-1ZM7 6a1 1 0 0 0-1 1v3a1 1 0 0 0 1 1h3a1 1 0 0 0 1-1V7a1 1 0 0 0-1-1H7Zm1 3V8h1v1H8Z"
+            clip-rule="evenodd"
+          />
+        </svg>
+        <span>Posts</span>
+      </a>
+    </aside>
+    <div id="admin/dashboard">
+      {% block panel %}
+      {% endblock panel %}
+    </div>
+  </div>
+  <link
+    rel="stylesheet"
+    href="https://unpkg.com/chartist@0.11.4/dist/chartist.min.css"
+  />
+  <script>
+    function createNewCounter(element, start, end, duration) {
+      let startTime = null;
+      const range = end - start;
+
+      function easeInOutCubic(t) {
+        t /= duration / 2;
+        if (t < 1) return (range / 2) * t * t * t + start;
+        t -= 2;
+        return (range / 2) * (t * t * t + 2) + start;
+      }
+
+      function step(timestamp) {
+        if (!startTime) startTime = timestamp;
+        const progress = timestamp - startTime;
+        const value = easeInOutCubic(progress);
+        element.textContent = Math.round(value);
+        if (progress < duration) {
+          requestAnimationFrame(step);
+        } else {
+          element.textContent = end;
+        }
+      }
+      requestAnimationFrame(step);
+    }
+    (() => {
+      const counters = document.querySelectorAll(".counter");
+      for (const counter of counters) {
+        createNewCounter(counter, 0, parseInt(counter.innerText), 4500);
```

Comment:
This animation is cool but it feels too slow. I want to see this information pretty immediately when I open the dashboard. Functionally the admin tools don't *need* any fancy animations at all but I do kind of like it for flavor. Reduce to 1 second maybe?